src/Templates/src/templates/maui-mobile/PageModels/ManageMetaPageModel.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections.ObjectModel;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using MauiApp._1.Data;
+using MauiApp._1.Models;
+using MauiApp._1.Services;
+
+namespace MauiApp._1.PageModels;
+
+public partial class ManageMetaPageModel : ObservableObject
+{
+	private readonly CategoryRepository _categoryRepository;
+	private readonly TagRepository _tagRepository;
+
+	[ObservableProperty]
+	private ObservableCollection<Category> _categories = [];
```

Comment:
why not use the property directly? Normally, when working with collections you don't want to allow external source to mutate it, and generated code will have a public setter.

```suggestion
	public ObservableCollection<Category> Categories { get; }= [];
```

Reply:
I think we'd also need to modify how the data is loaded here. 

Tags and Categoires are set after the view has been bound

![image](https://github.com/user-attachments/assets/dbcc5ea2-a6b4-4627-a9f2-21a71816b3df)

So in this scenario we'd need to init it in the ctor and then add them individually to maintain behavior

Reply:
Leave this in place for now.